templates/zerver/app/index.html

```diff
@@ -222,6 +222,11 @@
         <div class="alert-content"></div>
         <div class="exit"></div>
     </div>
+    <div class="alert alert_sidebar alert-error home-error-bar" id="found-missing-unreads">
+        <div class="exit"></div>
+        <strong class="message">{{ _('This conversation also has older unread messages.') }}</strong>
+        <a id="missing-unreads-jump-to-first-unread">{{ _('Jump to first unread.') }}</a>
+    </div>
```

Comment:
@sayamsamal how close as you to having a modernized version of this error banner component available? I'd really like to use Vlad's designs rather than this ancient thing designed for networking errors.

Reply:
Just finishing up with the final touches on #32852, which should be ready for testing tomorrow, and I'll make sure to follow-up with active feedback implementation. And #32791 also needs to be merged as a pre-requisite. With these two merged, we will be ready to use the modern banner components on this one.